apps/sshtosnmp.c

```diff
@@ -124,21 +137,35 @@ main(int argc, char **argv) {
     {
         struct msghdr msg = { 0 };
         struct iovec iov = { buf, buf_len };
-        struct ucred ouruser = { 0 };
         struct cmsghdr *cmsg;
-	char cmsgbuf[CMSG_SPACE(sizeof(ouruser))];
+	char cmsgbuf[2*4096];
+#if defined(SCM_CREDENTIALS)
+	struct ucred ouruser = { 0 };
```

Comment:
If we know that code is built for Linux, `{ 0 }` can be simplified into `{}`. Although this is a C23 feature, Linux C compilers support this feature since considerable time.

Reply:
But do we know that? Others might implement SCM_CREDENTIALS?

Reply:
That's fair. Let's keep `{ 0 }`.